assets/xml/objects/object_big_fwall.xml

```diff
@@ -1,6 +1,7 @@
 ﻿<Root>
+    <!-- Big fire wall used during Garo's encounters -->
     <File Name="object_big_fwall" Segment="6">
-        <Texture Name="object_big_fwall_Tex_000000" OutName="tex_000000" Format="i8" Width="32" Height="64" Offset="0x0" />
-        <DList Name="object_big_fwall_DL_0009A0" Offset="0x9A0" /> <!-- Original name is "big_firewall_modelT" -->
+        <Texture Name="gBigFireWallTex" OutName="big_firewall" Format="i8" Width="32" Height="64" Offset="0x0" />
+        <DList Name="gBigFireWallDL" Offset="0x9A0" /> <!-- Original name is "big_firewall_modelT" -->
```

Comment:
Maybe others can chime in (@tom-overton in particular since they're currently looking at Garo documentation), but I think `BigFireWall` could mean a few different things? So having Garo in the name would be helpful:
`gGaroFireWall`?
It's also a circular fire wall right? So I wonder if that could be put in the name somehow

Reply:
Yeah, from what I can tell, this is exclusively used for the ring of fire associated with the Garo fight. I think if you wanted to avoid putting "Garo" in the name, you could call it `gRingOfFireDL` or something like that; while it is tied to Garos in the final game, it's not *strictly* tied to them, since you could use it for something else. But `gGaroFireWall` or `gGaroRingOfFire` also works too; I'm not super picky.